BlueprintUI/Tests/BlueprintViewAdditions.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  BlueprintViewAdditions.swift
+//  BlueprintUI-Unit-Tests
+//
+//  Created by Kyle Van Essen on 2/21/20.
+//
+
+import UIKit
+
+@testable import BlueprintUI
+
+/**
+ Testing additions to help verify native view construction and management.
+ */
+extension BlueprintView
+{
+    internal var rootNativeElementViews : [UIView] {
+        self.rootNativeNodeView?.subviews ?? []
+    }
+    
+    private var rootNativeNodeView : RootNativeNodeView? {
```

Comment:
Make this hard fail – we control this, so it should not return a optional if we can't find the view.

Reply:
Actually, wait, no, this doesn't make sense – if the layout hasn't happened yet, there's no root note, so it could be nil. Leave as is.